R/shiny-driver.R

```diff
@@ -1,14 +1,17 @@
 #' Class to manage a shiny app and a `phantom.js` headless browser.
 #'
 #' @description
-#' This class starts a Shiny app in a new R session, along with a `phantom.js`
-#' headless browser that can be used to simulate user actions. This provides
-#' a full simulation of a Shiny app so that you can test user interactions
-#' with a live app.
+#'
+#' This class connects a `phantom.js` headless browser to a shiny app to enable
+#' testing the effect of user interactions with a live app.  Specify `url` to
```

Comment:
I think the path clause should come first since that's what most people will do.